cmd/oras/root/blob/fetch.go

```diff
@@ -157,6 +161,11 @@ func (opts *fetchBlobOptions) doFetch(ctx context.Context, src oras.ReadOnlyTarg
 			}
 		}()
 		writer = file
+	} else {
```

Comment:
Seems like these changes would be avoided with a validation in CheckArgs

Reply:
@TerryHowe is right, this validation can be done after all options are parsed via resetting `common.TTY` when needed.

Reply:
updated